ddtrace/tracer/tracer_test.go

```diff
@@ -187,6 +187,8 @@ func TestTracerStartSpan(t *testing.T) {
 			ext.PriorityAutoReject,
 			ext.PriorityAutoKeep,
 		}, span.Metrics[keySamplingPriority])
+		// A span is not measured unless made so specifically
+		assert.NotEqual(span.Meta[keyMeasured], "1")
```

Comment:
This assertion is incorrect because now we're setting `float64` value of `1`. Let's instead remove the comment and do:
```go
_, ok := span.Meta[keyMeasured]
assert.False(ok)
```